test/documentation/seqan3.css

```diff
@@ -0,0 +1,131 @@
+body {
+    background-color: #CCC;
+    color: black;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+}
+
+div, p, dl {
+    font-size: 13pt;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+    line-height: 30px;
+}
+
+table {
+    font-size: 12pt;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
```

Comment:
Why all the Apple fonts? But I guess we can adapt this later...

Reply:
These are the default fonts from github. I just copy and pasted them, and trust that Github chose them wisely.